android/manifest

```diff
@@ -2,17 +2,17 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 1.0.0
-apiversion: 2
+version: 1.1.0
```

Comment:
Maybe it would be better to bump the major number to 2.0.0 here, since it's a breaking change, incompatible with previous Titanium versions